front/app/components/UI/Error/messages.ts

```diff
@@ -231,4 +231,62 @@ export default defineMessages({
     id: 'app.errors.category_name_taken',
     defaultMessage: 'A category with this name already exists',
   },
+  bulk_import_ideas_maximum_ideas_exceeded: {
+    id: 'app.errors.bulk_import_ideas_maximum_ideas_exceeded',
+    defaultMessage: 'The maximal amount of {value} ideas has been exceeded.',
```

Comment:
From the internet: 'Maximum' is the greatest; 'maximal' is tending towards the greatest. I would also leave out the 'amount'. So I think it should be:

```suggestion
    defaultMessage: 'The maximum of {value} ideas has been exceeded.',
```